tracker_automations/continuous_manage_queues/continuous_manage_queues.sh

```diff
@@ -72,21 +72,11 @@ movemax=${movemax:-3}
 lastweekdate=${lastweekdate:-$(date -d "${releasedate} -7day" +%Y-%m-%d)}
 dryrun=${dryrun:-}
 
-# Verify that $releasedata has a correct YYYY-MM-DD format
-if [[ ! ${releasedate} =~ ^[0-9]{4}-[0-9]{2}-[0-9]{2}$ ]]; then
-    echo "ERROR: \$releasedate. Incorrect YYYY-MM-DD format detected: ${releasedate}"
-    exit 1
-fi
-
-# Verify that $lastweekdate has a correct YYYY-MM-DD format
-if [[ ! ${lastweekdate} =~ ^[0-9]{4}-[0-9]{2}-[0-9]{2}$ ]]; then
-    echo "ERROR: \$lastweekdate. Incorrect YYYY-MM-DD format detected: ${lastweekdate}"
-    exit 1
-fi
-
 # Today
 nowdate=$(date +%Y%m%d)
 
+run_param_validation $releasedate $lastweekdate
```

Comment:
As far as you're relying on this script being executed with `-e` to exit on functions returning non-zero... it's a risk if, someday, that's changed for some reason (say debugging or whatever). So I would recommend to do some of these, to ensure that we don't forget about the script requiring `-e`.
- Or you add a comment in the line saying that this script won't work if changed.
- Or you change the `return 1` statements by direct `exit 1` calls.
- Or you capture the results of the function and add a "guard" condition to, in case it returns non-zero, it exits.

As said, the current code is correct, but we need to ensure that the behavior is the same as much as possible, no matter of the exit-on-error setting.

Reply:
Thanks, Eloy! Good catch! I used `exit 1` for the failed validation cases.